packages/components/src/components/primitives/Icons/Icons.tsx

```diff
@@ -15,7 +15,7 @@ import * as Tabler from './TablerIcons';
 import { IconProps } from './types';
 
 const withIcon = (Component: React.FC, props: IconProps) => {
-  const styles = useStyleConfig('Icon') as Record<
+  const styles = useStyleConfig('Icon', { size: props?.size || 'm' }) as Record<
```

Comment:
Could this size be defined in the `defaultProps` of the theme level? `packages/components/src/theme/ComposerTheme/Components/Icon.ts`